src/.env.sample

```
@@ -192,4 +192,5 @@ SERVICE_NAME='MentoringService'
 
 #To disable logs for healthCheck
 HEALTH_CHECK_DEBUG_MODE='false'
-EVENT_TENANT_KAFKA_TOPIC='tenantEvent'
\ No newline at end of file
+EVENT_TENANT_KAFKA_TOPIC='tenantEvent'
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)